pandas/tests/groupby/test_quantile.py

```diff
@@ -232,3 +232,11 @@ def test_groupby_quantile_nullable_array(values, q):
 
     expected = pd.Series(true_quantiles * 2, index=idx, name="b")
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("q", [0.5, [0.0, 0.5, 1.0]])
```

Comment:
can you also try with an empty frame that has the same dtypes, that should raise?

Reply:
It does not raise on an empty frame, but instead returns an empty frame with the same columns of the valid dtypes, in the case of the test `Columns: [b]`.